### ISSUE TITLE
Fail to reproduce the results

### ISSUE THREAD

**Issue by uiyo** [naver-ai/DenseDiffusion#8]
Thanks for this excellent work!

Unfortunately, I encountered some difficulties while attempting to replicate the results from your paper. Would you be able to provide additional details regarding the methodology and experimental setup to help me better understand and reproduce your research findings?

![image](https://github.com/naver-ai/DenseDiffusion/assets/16344469/46418b77-b192-4b51-bcb0-19a798d74e1b)

 Here is my hyperparameters, I tried to make the mask more controllable by increasing the value of w^s and w^c, however, none of them are useful.
![image](https://github.com/naver-ai/DenseDiffusion/assets/16344469/3caf4898-6ddb-4b83-bd35-34131a53a407)

**Comment by YunjiKim** [comment]
can I ask which version of SD are you using?

**Comment by uiyo** [comment]
sd v1.5

**Comment by YunjiKim** [comment]
we recommend using the default values.
generally, results are not good if values after the modulation deviate too much from the original ones.
please refer to paragraph "Value-range Adaptive Attention Modulation" of section 3.2 in the paper of this work.

**Comment by uiyo** [comment]
Default setting cannot provide satisfying results though :(, not so sure why this happen
![image](https://github.com/naver-ai/DenseDiffusion/assets/16344469/5df7e7d5-3cad-46e1-9110-6a77bc30e34e)

**Comment by YunjiKim** [comment]
it works okay on my device with the layout condition similar to yours ... 
<img width="1319" alt="screenshot" src="https://github.com/naver-ai/DenseDiffusion/assets/22008183/0c5ccd6d-eb45-43f5-a635-28f533f47550">

could you try with a refined full text in (3), and if it is still not working,
please share your gradio_app.py file with me so that I can check the problem.

**Comment by uiyo** [comment]
Still not working, I tried refined full text in (3), here is the results:
![image](https://github.com/naver-ai/DenseDiffusion/assets/16344469/983fd2f2-46ba-41f6-9fbb-39650742231b)

I merely changed two lines in gradio_app.py:
```
line 18: os.environ["GRADIO_TEMP_DIR"] = 'my dir'
line 69: use_auth_token='my token'

```

**Comment by YunjiKim** [comment]
I updated gradio_app.py file with some examples.
if you keep failing and if it is not a burden for you, could you try with redownloaded SD v1.5?

**Comment by uiyo** [comment]
I might have figured out why.
Also I encodered this issue #4 , my diffusers version is 0.19.3, and tried 0.20.1 too, still this error.
To make it work I did:
In pipline_stable_diffusion.py, change:
![image](https://github.com/naver-ai/DenseDiffusion/assets/16344469/be2a11c8-1d51-4b08-9a61-850bcf36d55a)
to 
![image](https://github.com/naver-ai/DenseDiffusion/assets/16344469/6e5f468e-d8ec-44f8-8bef-f69c0a98a0e2)
Seems this modification will essentially bypass the NSFW filter, maybe that's the reason I failed to produce the results.

Then I downgraded the diffusers to 0.12.1, redownloaded the sd v1.5.
Here is my results :), the results look fine:
![image](https://github.com/naver-ai/DenseDiffusion/assets/16344469/966ff6dd-5a9d-48d9-ad92-1a6c91b73979)
